doc/mgr/orchestrator.rst

```diff
@@ -273,11 +273,72 @@ Placement Specification
 
 In order to allow the orchestrator to deploy a *service*, it needs to
 know how many and where it should deploy *daemons*. The orchestrator 
-defines a placement specification:
+defines a placement specification that can either be passed as command line arguments.
 
-* To deploy three *daemons*, simply specify the count: ``3``
-* To deploy *daemons* on specific hosts, specify the host names: ``host1 host2 host3``
-* To deploy *daemons* on a subset of hosts, also specify the count: ``2 host1 host2 host3``
+Explicit placements
+^^^^^^^^^^^^^^^^^^^
+
+Daemons can be explictly placed on hosts by simply specifying them::
+
+    orch apply prometheus "host1 host2 host3"
+    
+or in yaml::
+  
+    service_type: prometheus
+    placement:
+      hosts: "host1 host2 host3"
+     
+MONs and other services may require some enhanced network specifications::
+
+  orch daemon add mon myhost:[v2:1.2.3.4:3000,v1:1.2.3.4:6789]=name
+  
+Where ``[v2:1.2.3.4:3000,v1:1.2.3.4:6789]`` is the netowrk address of the monitor
+and ``=name`` will specify the name of the new monitor.
+
+Placement by Labels
+^^^^^^^^^^^^^^^^^^^
+
+Daemons can be explictly placed on hosts that match a specifc label::
+
+    orch apply prometheus "label:mylabel"
+
+or in yaml::
+
+    service_type: prometheus
+    placement:
+      label: "mylabel"
+
+
+Placement on all hosts
+^^^^^^^^^^^^^^^^^^^^^^
+
+Daemons can be explictly placed on all hosts as well::
+
+    orch apply prometheus "all:true"
+
+or in yaml::
+
+    service_type: prometheus
+    placement:
+      all_hosts: true
+
+
+Setting a limit
+^^^^^^^^^^^^^^^
+
+By specifying ``count``, only that number of daemons will be created::
+
+    orch apply prometheus 3
+    
+To deploy *daemons* on a subset of hosts, also specify the count::
+
+    orch apply prometheus "2 host1 host2 host3"
```

Comment:
could also add the converse, e.g., "3 host1 host2"

Reply:
fixed.